### PR TITLE
avoid installing postgresql-libs v8.4

### PR DESCRIPTION
**Pull request by mattjdnv**
The postgresql tests keep failing!
1. refs #140

I added postgresql9.3 and removed v9.2.

### CODE REVIEW

File: BuildArchive.sh [hunk 1]
```diff
@@ -26,12 +26,19 @@ make -j $((`nproc` + 2)) tmp/hoot-deps
 
 # init and start Postgres
 cd /tmp
-PG_VERSION=9.2
-sudo service postgresql-$PG_VERSION initdb
-sudo service postgresql-$PG_VERSION start
+# init and start Postgres
+PG_SERVICE=$(ls /etc/init.d | grep postgresql- | sort | tail -1)
```
Comment (sputnam) [inline BuildArchive.sh]: Will this work on both Ubuntu 14.04 and CentOS 6.7?

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 
 # Default branch of hoot to build
-GIT_COMMIT?=origin/develop
+#GIT_COMMIT?=origin/develop
+GIT_COMMIT=b146af25190210dab38d01e5b
```
Comment (sputnam) [inline Makefile]: I'm assuming we want this set back to `origin/develop`?

File: src/SPECS/hootenanny.spec.in [hunk 3]
```diff
@@ -18,7 +18,8 @@ BuildRequires:  git glpk-devel graphviz libicu-devel
 BuildRequires:  hootenanny-words = 1.0.0
 BuildRequires:  libspatialite-devel
 BuildRequires:  log4cxx-devel nodejs-devel opencv-devel
-BuildRequires:  postgresql-devel >= 9.1
+#BuildRequires:  postgresql-devel >= 9.1
```
Comment (sputnam) [inline src/SPECS/hootenanny.spec.in]: Remove old line?

File: src/SPECS/hootenanny.spec.in [hunk 4]
```diff
@@ -18,7 +18,8 @@ BuildRequires:  git glpk-devel graphviz libicu-devel
...
+BuildRequires:  postgresql93-devel
```
Comment (sputnam) [inline src/SPECS/hootenanny.spec.in]: Do we need to handle the case where we are doing an upgrade to an existing install running postgresql 9.2?

File: src/SPECS/hootenanny.spec.in [hunk 5]
```diff
@@ -592,9 +603,11 @@ Requires:   %{name}-core-devel-deps = %{version}-%{release}
 Requires:   apache-maven
 Requires:   nodejs
 Requires:   npm
-Requires:   postgresql-server >= 9.1
-Requires:   postgresql-contrib >= 9.1
-Requires:   postgis2_92 >= 2.1
+#Requires:   postgresql-server >= 9.1
```
Comment (sputnam) [inline src/SPECS/hootenanny.spec.in]: Remove old lines?

File: src/SPECS/hootenanny.spec.in [hunk 6]
```diff
@@ -592,9 +603,11 @@ Requires:   %{name}-core-devel-deps = %{version}-%{release}
...
+#Requires:   postgresql-contrib >= 9.1
```
Comment (sputnam) [inline src/SPECS/hootenanny.spec.in]: Remove old line?

File: BuildArchive.sh (hunk 1 above)
Comment (mattjdnv) [inline BuildArchive.sh]: I don't think so. Postgres has a version number on Centos and it doesn't on Ubuntu.

File: Makefile (hunk 2 above)
Comment (mattjdnv) [inline Makefile]: Sorry, That was set for testing.

File: src/SPECS/hootenanny.spec.in (hunk 3 above)
Comment (mattjdnv) [inline src/SPECS/hootenanny.spec.in]: Yes.

File: src/SPECS/hootenanny.spec.in (hunk 4 above)
Comment (mattjdnv) [inline src/SPECS/hootenanny.spec.in]: Yes. I figured that Damian was already running 93 so it wouldn't be a problem.

I can go back to 92 and I have another strategy for avoiding the postgresql-libs 8.4 install that I'm going to test.

File: src/SPECS/hootenanny.spec.in (hunk 5 above)
Comment (mattjdnv) [inline src/SPECS/hootenanny.spec.in]: OK

File: src/SPECS/hootenanny.spec.in (hunk 6 above)
Comment (mattjdnv) [inline src/SPECS/hootenanny.spec.in]: OK

File: src/SPECS/hootenanny.spec.in (hunk 4 above)
Comment (sputnam) [inline src/SPECS/hootenanny.spec.in]: Ah, good point about already running 9.3.  I'm all about consolidating versions so if we think 9.3 is the way to go, then I'm all for it.  Just want to make sure we've thought about any impacts to upgrading.

**Comment by mattjdnv** [comment]
please build rpms

**Comment by sputnam** [comment]
please build rpms

File: Makefile [hunk 7]
```diff
@@ -1,7 +1,9 @@
 
 # Default branch of hoot to build
-#GIT_COMMIT?=origin/develop
-GIT_COMMIT=b146af25190210dab38d01e5b
+GIT_COMMIT?=origin/develop
+# Branch #1560 for testing
```
Comment (sputnam) [inline Makefile]: Recommend removing commented out test lines before merge.

File: Makefile [hunk 8]
```diff
@@ -69,6 +71,8 @@ vagrant-clean:
 	rm -f src/tmp/*-install
 
 vagrant-test:
+	# Adding this in so we always have a clean test VM.
+	cd test && vagrant halt && vagrant destroy -f
```
Comment (sputnam) [inline Makefile]: This will be a nice change.

File: test/Vagrantfile [hunk 9]
```diff
@@ -34,6 +34,7 @@ $provisionSh = <<-SHELL
 
     sudo yum -y -q update --exclude=puppet*
     echo "Install Hootenanny"
+#     sudo yum -y -v install hootenanny-autostart
```
Comment (sputnam) [inline test/Vagrantfile]: Do we want the verbose output since this a test run?  If it's not useful then I'm fine with the change.

**Pull request merged**